### DISCUSSION TITLE
Image repo settings for kube-rbac-proxy

### DISCUSSION THREAD

**Discussion by itroyano** [kubernetes-sigs/kubebuilder#2375] [Support] [answered]
Hi,

When generating with Operator SDK, the image for kube-rbac-proxy sidecar is set to be pulled from gcr.io.
is there a flag or annotation to change the repo when generating?
I'd like to use kube-rbac-proxy from quay.io or my local repo.

**Comment by camilamacedo86** [comment, marked as answer]
Then, you can update the default scaffold of your project to looking for this image from the place that please you. See that it is under the config/ directory. You can just use find to check all references and replace them.

I hope that answered your question,
I am closing it as answered. However, please feel free to re-open if you need.
